lib/esp/resources/suppression.rb

```diff
@@ -98,7 +98,7 @@ def deactivate
       patch(:deactivate).tap do |response|
         load_attributes_from_response(response)
       end
-    rescue ActiveResource::BadRequest, ActiveResource::ResourceInvalid, ActiveResource::UnauthorizedAccess => error
+    rescue ActiveResource::BadRequest, ActiveResource::ResourceInvalid, ActiveResource::UnauthorizedAccess, ActiveResource::ForbiddenAccess => error
```

Comment:
It looks like there is also a spot in Custom Signature Definitions that needs this change.